### PR TITLE
[do not merge] Publish jobs as coalesced when reason superseded

### PR DESCRIPTION
**Pull request by gregarndt**

### CODE REVIEW

File: src/handler.js [hunk 1]
```diff
@@ -39,6 +39,10 @@ function resultFromRun(run) {
         return 'canceled';
       }
 
+      if (run.reasonResolved === 'superseded') {
+        return 'coalesced';
```
Comment (camd) [inline src/handler.js]: Hrm, I could have sworn I submitted a review to you on this already.  Sheesh, so sorry.  We have actually decided to migrate to your term ``superseded`` internally in Treeherder.  I have merged my PR that uses (and expects) that term as a result.  So if you'd switch this to ``superseded`` here and in your version of the ``yml`` then we'll be good to go.  

My PR is on stage now, and should be on prod Tuesday next week, I expect.

File: schemas/pulse-job.yml [hunk 2]
```diff
@@ -171,13 +171,15 @@ properties:
...
       unknown: When the job is not yet completed
+      coalesced: When a job has been superseded by another job
     type: "string"
     enum:
       - success
       - fail
       - exception
       - canceled
       - unknown
+      - coalesced
```
Comment (camd) [inline schemas/pulse-job.yml]: change to ``superseded``

Comment (gregarndt) [reply schemas/pulse-job.yml]: should be fixed in 4745646

**Comment by gregarndt** [comment]
Please let me know when the TH side lands on prod, then we can land this.  Was the list of coalesced job guids also removed from the schema?

**Comment by camd** [comment]
The Treeherder changes have all been merged and pushed to production now.  So you should be good to go.
  :)

**Comment by gregarndt** [comment]
doh! good catch, changed in 8044066d746a54d18db04d52cde40de1ae01267f

**Comment by gregarndt** [comment]
If travis likes this, I can merge and deploy

**Pull request merged**